### PR TITLE
Updating symlink and directory creation guidance

### PR DESCRIPTION
**Pull request by bergerhoffer**
Main only - this is just a guideline clarification

### CODE REVIEW

File: contributing_to_docs/doc_guidelines.adoc [hunk 1]
```diff
@@ -207,23 +207,29 @@ Try to shorten the file name as much as possible _without_ abbreviating importan
...
 
-If you create a directory with a multiple-word name, separate each word with an underscore, for example `backup_and_restore`.
+When creating a new directory, follow these guidelines:
 
-Do not create or rename a top-level directory in the repository and topic map without checking with the docs program manager first.
+* Use underscores to separate multiple-word directory names, for example, `backup_and_restore`.
+* Create the symbolic links for the `images`, `modules`, `snippets` and `_attributes` directories. See xref:symbolic-links[Symbolic links] for instructions.
+* Use the CLI (`mkdir <directory_name>`) instead of an editor, to avoid potential syncing issues.
+* Do not copy another directory or its symbolic links when creating the new directory, to avoid issues with the symbolic links.
+* Do not create or rename a top-level directory in the repository and topic map without checking with the docs program manager first. If a book is being created or modified, there are changes on the Customer Portal that must also be made.
```
[the remark this answers is not in the document]
Comment (bergerhoffer) [reply contributing_to_docs/doc_guidelines.adoc:216]: @aireilly @rohennes This shouldn't be an error. We updated the SSG in January to allow "Customer Portal": https://redhat-documentation.github.io/supplementary-style-guide/#january-2024

[the remark this answers is not in the document]
Comment (aireilly) [reply contributing_to_docs/doc_guidelines.adoc:216]: https://github.com/redhat-documentation/vale-at-red-hat/pull/764

File: contributing_to_docs/doc_guidelines.adoc [hunk 2]
```diff
@@ -236,16 +242,16 @@ $ ln -s <target_directory> <link_name>
...
+[openshift_cli ~]$ ln -s ../../modules/ modules
+[openshift_cli ~]$ ln -s ../../snippets/ snippets
+[openshift_cli ~]$ ln -s ../../_attributes/ _attributes
 ----
 +
 Be sure to adjust the number of levels to back up (`../`) depending on how deep your directory is.
 
-If you accidentally create an incorrect link, you can remove that link by using `unlink <link_name>`.
-====
+If you accidentally create an incorrect link, you can remove that link by using `unlink <link_name>`. For example, to unlink the `images-typo` symlink in the `cli_reference/openshift_cli` directory, run `unlink images-typo`.
```
Comment (ShaunaDiaz) [inline contributing_to_docs/doc_guidelines.adoc:254]:
```suggestion
If you accidentally create an incorrect link, you can remove that link by using `unlink <link_name>`. For example, to unlink the `images` symlink in the `cli_reference/openshift_cli` directory, run `unlink images`.
```
Not sure what the "typo" meant here.

File: contributing_to_docs/doc_guidelines.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (bergerhoffer) [reply contributing_to_docs/doc_guidelines.adoc:216]: Thanks!

File: contributing_to_docs/doc_guidelines.adoc (hunk 2 above)
Comment (bergerhoffer) [reply contributing_to_docs/doc_guidelines.adoc:254]: I was doing that to show that you made a mistake in creating the link. Could be in the name of the symlink or could be in the directory path you chose.

But if what I have here is more confusing than helpful, I'll remove it

**Review by sheriff-rh — APPROVED** [review]
Changes look good, thank you Andrea!

**Comment by dfitzmau** [comment]
Nice work. Andrea. LGTM.

**Pull request merged**